dynatrace-aws-logs.sh

```diff
@@ -150,6 +151,69 @@ arguments:
   aws cloudformation describe-stacks --stack-name "$STACK_NAME" --query "Stacks[0].Outputs"
   ;;
 
+"delete")
+
+  function print_help_delete {
+    printf "
+usage: dynatrace-aws-logs.sh delete [--stack-name STACK_NAME]
+
+arguments:
+  -h, --help            show this help message and exit
+  --stack-name STACK_NAME
+                        Optional. The name for the CloudFormation stack in which the resources will be deployed. This defaults to \"$DEFAULT_STACK_NAME\"
+"
+  }
+
+  function print_params_delete {
+    echo
+    echo "Deletion script will use following parameters:"
+    echo "STACK_NAME=\"$STACK_NAME\""
+  }
+
+  function ensure_param_value_given {
+    # Checks if a value ($2) was passed for a parameter ($1). The two OR'ed conditions catch the following mistakes:
+    # 1. The parameter is the last one and has no value
+    # 2. The parameter is between other parameters and (as it has no value) the name of the next parameter is taken as its value
+    if [ -z $2 ] || [[ $2 == "--"* ]]; then echo "Missing value for parameter $1"; print_help_delete; exit 1; fi
+  }
+
+  while (( "$#" )); do
+    case "$1" in
+
+      "--stack-name")
+        ensure_param_value_given $1 $2
+        STACK_NAME=$2
+        shift;shift;
+      ;;
+
+      "-h")
+        print_help_delete
+        shift; exit 0
+      ;;
+      "--help")
```

Comment:
tip - you can use `"-h" | "--help")` in case to not duplicate code

Reply:
Thanks. Will update.

Reply:
Done.